students/julimatos.Rmd

```diff
@@ -0,0 +1,43 @@
+##### NEED TO FIX IMAGE, AND READ CSV
+
+
+---
+title: "RMarkdown Assignment"
+output:
+  html_document:
+    toc: true
+    toc_depth: 2
+---
+
+ ## Content
+  
+  Our [group project](http://sbfire.weebly.com/) is an *assessment* of the benefits of fire mitigation programs cities in the **wildland urban interface**, using Santa Barbara as a case study. We have already distributed a survey to the residents of the City of Santa Barbara's Wildland Fire Suppression Assessment District in order to gauge the attitudes of the residents toward the program. We have also ran fire modeling programs to determine the effectiveness of the vegetation treatments of the program. The final component of our project will be looking at the transferability of the program, and we are trying to determine what other cities in California may benefit from a similar program. 
+  
+  The picture below shows the extent of the Wildland Fire Suppression Assessment District in the City of Santa Barbara.
+  ![](/Users/julianamatos/Desktop/WFSAD.png)
```

Comment:
Hi @julimatos, the above line:

```md
![](/Users/julianamatos/Desktop/WFSAD.png)
```
is file is an "absolute" path that works on your computer but does not exist in Github. Please move `/Users/julianamatos/Desktop/WFSAD.png` to `images/julimatos_WFSAD.png` and update path in Rmd above to similarly use "relative" location to your `students/julimatos.Rmd`.